blocks/media-gallery/media-gallery.css

```diff
@@ -0,0 +1,156 @@
+.media-gallery-wrapper {
+    width: 100vw;
+}
+
+.media-gallery {
+    ul {
+        list-style-type: none;
+        display: grid;
+        grid-gap: 1rem;
+        width: calc(100vw - 4rem);
+        margin: auto;
+        padding: 0;
+    }
+    p {
+        margin: 0;
+    }
+    picture {
+        display: block;
+        aspect-ratio: 16/9;
+        overflow: hidden;
+        position: relative;
+    }
+    picture:after {
+        --gradient-angle: to top;
+        --gradient-angle-m: to top;
+        --gradient-transition: 60%;
+        content: "";
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        inset-block-end: 0;
+        inset-inline-start: 0;
+        background: linear-gradient(to top, rgba(53, 53, 53, 0.8), transparent 60%);
+    }
+    .cards-card-body {
+        position: relative;
+    }
+    .card-content {
+        position: absolute;
+        z-index: 1;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        padding: 24px;
+        padding-top: 0;
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+    }
+    .card-title {
+        pointer-events: none;
+        color: white;
+        font-weight: 400;
+        font-family: var(--calcite-sans-family);
+        line-height: 1.2;
+        font-size: var(--font-2);
+        text-align: start;
+        margin: 0;
+    }
+    .card-description {
+        display: none;
+        pointer-events: none;
+        color: white;
+        font-weight: 400;
+        font-family: var(--calcite-sans-family);
+        text-align: start;
+        font-size: clamp(.8rem,1vw,1rem);
+        line-height: 1.2;
+        max-height: 2.4; /* 2 lines */
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 2;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        margin-block-end: 0;
+    }
+    img {
+        width: 100%;
+        aspect-ratio: 16/9;
+        object-fit: cover;
+    }
+    .card-content, img {
+        transition: 0.2s ease;
+    }
+    .cards-card-body:is(:hover,:focus,:focus-within) .card-content {
+        box-shadow: 0 0 0 10px inset rgba(255, 255, 255, 0.8);
+    }
+    .cards-card-body:is(:hover,:focus,:focus-within) img {
+        transform: scale(1.08);
+    }
+    .start-button {
+        margin: 0;
+        margin-bottom: 8px;
+        width: 44px;
+        height: 44px;
+        color: rgb(21, 21, 21);
+        background-color: rgb(248, 248, 248);
```

Comment:
Can you use hex codes for the color codes? (mostly for uniformity)

Reply:
Ideally we should use the Calcite color variables. They have our approved hex colors and work in light/dark modes. 
 --calcite-ui-brand
    --calcite-ui-brand-hover
    --calcite-ui-brand-press
    --calcite-ui-background-1
    --calcite-ui-foreground-1
    --calcite-ui-foreground-2
    --calcite-ui-foreground-3
    --calcite-ui-text-1
    --calcite-ui-text-2
    --calcite-ui-text-3
    --calcite-ui-text-inverse
    --calcite-ui-text-link
    --calcite-ui-border-1
    --calcite-ui-border-2
    --calcite-ui-border-3